portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,7 +54,7 @@ public void init() {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     /* When the pointer exceeds the size of the facts list (or somehow becomes
-     * negative), reset currQuestionIndex and reshuffle the facts list */
+     * negative), reset currQuestionIndex and reshuffle the facts list. */
```

Comment:
I recommend all comments end with periods so that they are consistent with longer ones that require them, like this. This is OK for now, but moving forward keeping them consistent is preferred.